mbed_lstools/lstools_base.py

```diff
@@ -267,6 +267,12 @@ def __init__(self, **kwargs):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "C002": "VK_RZ_A1H",
+        "C030": "C030-U201",
+        "C031": "C030-N211",
+        "C032": "C030-R404M",
+        "C033": "C030-R410M",
+        "C034": "C030-S200",
+        "C035": "C030-R3121",
```

Comment:
@RobMeades Would it be possible to replace the `-` (hyphens) with `_` underscores? It looks like all other platforms use underscores.